src/VisualStudio/IntegrationTest/IntegrationTests/AbstractEditorTest.cs

```diff
@@ -66,5 +68,14 @@ protected ClassifiedToken[] GetLightbulbPreviewClassification(string menuText)
         {
             return VisualStudio.Editor.GetLightbulbPreviewClassification(menuText);
         }
+
+        protected string PrependLines(string input, string toAdd)
```

Comment:
Not sure if inheritance is the best place to expose this. Do we have some other string helpers type?